datastore/src/main/java/org/jboss/pnc/datastore/repositories/audited/AuditedBuildConfigurationRepository.java

```diff
@@ -0,0 +1,23 @@
+package org.jboss.pnc.datastore.repositories.audited;
+
+import org.hibernate.envers.AuditReader;
+import org.jboss.pnc.datastore.audit.impl.AbstractAuditRepository;
+import org.jboss.pnc.model.BuildConfiguration;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class AuditedBuildConfigurationRepository extends AbstractAuditRepository<BuildConfiguration, Integer> {
+
+    @Deprecated
```

Comment:
Ok, I'll implement it tomorrow.

On 04/08/2015 04:35 PM, Paul Gier wrote:
>
> In 
> datastore/src/main/java/org/jboss/pnc/datastore/repositories/audited/AuditedBuildConfigurationRepository.java 
> <https://github.com/project-ncl/pnc/pull/278#discussion_r27975120>:
>
> > @@ -0,0 +1,23 @@
> > +package org.jboss.pnc.datastore.repositories.audited;
> > +
> > +import org.hibernate.envers.AuditReader;
> > +import org.jboss.pnc.datastore.audit.impl.AbstractAuditRepository;
> > +import org.jboss.pnc.model.BuildConfiguration;
> > +
> > +import javax.enterprise.context.ApplicationScoped;
> > +import javax.inject.Inject;
> > +
> > +@ApplicationScoped
> > +public class AuditedBuildConfigurationRepository extends AbstractAuditRepository<BuildConfiguration, Integer> {
> > +
> > +    @Deprecated
>
> Deprecated makes it sound like it will be removed eventually, but I 
> guess there isn't a better tag to use to say that it just shouldn't be 
> used as public api. Can you add a javadoc comment to this constructor?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/project-ncl/pnc/pull/278/files#r27975120>.
>